gonvme_test.go

```diff
@@ -19,10 +19,11 @@ package gonvme
 import (
 	"encoding/json"
 	"fmt"
-	log "github.com/sirupsen/logrus"
 	"io/ioutil"
 	"strings"
 	"testing"
+
```

Comment:
remove the extra line

Reply:
Thanks @prablr79 for the feedback. Actually it is automatically getting added by the go formatter tool.